src/Widgets.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt;
+
+use Bolt\Snippet\QueueProcessor;
+use Bolt\Snippet\Target;
+use Bolt\Snippet\Zone;
+use Bolt\Widget\BaseWidget;
+use Bolt\Widget\BoltHeaderWidget;
+use Bolt\Widget\CanonicalLinkWidget;
+use Bolt\Widget\NewsWidget;
+use Bolt\Widget\SnippetWidget;
+use Bolt\Widget\WeatherWidget;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Response;
+use Tightenco\Collect\Support\Collection;
+use Twig\Environment;
+
+class Widgets
```

Comment:
IMO this class does too much - it should either register widgets or render them

Reply:
I'm not sure if it makes sense to split it.. Basically all it does is add items to the queue and pop them (in groups or one by one)

Reply:
This is something that can be added as @todo for further improvements.

Reply:
I've removed the `registerBoltWidgets` method from widgets as a start. It's not the right place.. Added a `@todo` as well.